examples/gno.land/p/demo/zteams/gno.mod

```diff
@@ -0,0 +1 @@
+module gno.land/p/demo/zteams
```

Comment:
Please run `gno mod tidy` to update the dependencies. Additionally, run `make fmt` to resolve CI errors.

Reply:
done, thanks for the tips, didn't know about `make fmt`